spi_nand_flash/src/nand.c

```diff
@@ -279,6 +279,8 @@ esp_err_t spi_nand_erase_chip(spi_nand_flash_device_t *handle)
                           end, TAG, "");
     }
 
+    xSemaphoreGive(handle->mutex);
+
```

Comment:
Alternatively, we can remove `return ESP_OK;` on the following line. Then we will go through the `end` label, releasing the mutex and returning ESP_OK.